CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1996,6 +1996,11 @@ public let DECL_NODES: [Node] = [
     parserFunction: "parseSourceFile",
     traits: ["WithStatements"],
     children: [
+      Child(
+        name: "Hashbang",
```

Comment:
Could you add some documentation for this child? Just briefly describe in which cases the shebang can exist and what it means in terms of Swift.